### PR TITLE
PGD CLI copy edits

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Updates to main landing page and release notes

### CODE REVIEW

**Comment by drothery-edb** [comment]
This replaces https://github.com/EnterpriseDB/docs/pull/2630 since we are using the heritage EDB product release note format.

File: product_docs/docs/pgd_cli/1/rel_notes/index.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+---
+
+PGD CLI is a tool to manage your BDR cluster. It allows you to run commands against BDR clusters. 
+
+The PGD CLI documentation describes the latest version of PGD CLI including minor releases and patches. The release notes in this section provide information on what was new in each release.
+
+
+| Version                  | Release Date | 
+| -----------------------  | ------------ | 
+| [1](epd_cli_1_rel_notes) | 2022 May 17  |
```
Comment (smaher-edb) [inline product_docs/docs/pgd_cli/1/rel_notes/index.mdx:13]: I think we should name the file as `pgd_cli_1_rel_notes` instead of `epd_cli_1_rel_notes`

**Review by smaher-edb — APPROVED** [review]
Except some typos and few changes in 'Output' format section other things LGTM.

**Comment by drothery-edb** [comment]
@smaher-edb thank you for your excellent comments. I have incorporated them. Do you want to look the changes over before I merge the PR to our main working branch?

**Comment by smaher-edb** [comment]
> merge the PR to our main working bran

Thanks. Changes LGTM.

**Pull request merged**